clients/automation/automation.go

```diff
@@ -46,18 +46,36 @@ const (
 	SchedulingRules
 )
 
-var resources = map[ResourceType]Resource{
-	Workflows:         {Path: "/platform/automation/v1/workflows"},
-	BusinessCalendars: {Path: "/platform/automation/v1/business-calendars"},
-	SchedulingRules:   {Path: "/platform/automation/v1/scheduling-rules"},
-}
+const (
+	errMsg          = "failed to %s automation resource of type %v: %w"
+	errMsgWithId    = "failed to %s automation resource of type %v with id %s: %w"
+	getOperation    = "get"
+	updateOperation = "update"
+	listOperation   = "list"
+	deleteOperation = "delete"
+	createOperation = "create"
+)
+
+var (
+	resources = map[ResourceType]Resource{
```

Comment:
As we move towards changing what a resource is, what is the feeling about splitting this into three separate resource in a future follow-up?